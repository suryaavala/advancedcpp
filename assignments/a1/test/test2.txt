1.0
2 add
3 mult
4 div
5 sub
6.0 sqrt