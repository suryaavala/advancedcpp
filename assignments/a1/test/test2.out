2 + 1.000 = 3.000
3 * 3.000 = 9.000
4 / 9.000 = 0.444
5 - 0.444 = 4.556
sqrt 6.000 = 2.449
